sdk/documentintelligence/azure-ai-documentintelligence/azure/ai/documentintelligence/_operations/_patch.py

```diff
@@ -27,6 +31,57 @@
 JSON = MutableMapping[str, Any]  # pylint: disable=unsubscriptable-object
 T = TypeVar("T")
 ClsType = Optional[Callable[[PipelineResponse[HttpRequest, HttpResponse], T, Dict[str, Any]], Any]]
+PollingReturnType_co = TypeVar("PollingReturnType_co", covariant=True)
+_FINISHED = frozenset(["succeeded", "canceled", "failed", "completed"])
+
+
+def _parse_operation_id(operation_location_header):
+    regex = "[^:]+://[^/]+/documentintelligence/.+/([^?/]+)"
+    return re.match(regex, operation_location_header).group(1)
+
+
+class AnalyzeDocumentLROPollingMethod(LROBasePolling):
```

Comment:
We need to also override `from_continuation_token` from LROBasePolling to ensure we use our custom polling method if a user decides to rehydrate the response from a continuation_token

Reply:
Added